routes/projects.rb

```diff
@@ -70,15 +70,21 @@ def self.registered(app)
         project.update(name:  params[:name] || project.name,
                        description: params[:description] || project.description,
                        active: params[:active] || project.active)
-        redirect "/projects/#{params[:id]}"
-      end
 
-      app.delete '/projects/:id/?' do
```

Comment:
Same with clients above, lets have an actual delete method, not patch, and look up the default id by name.